### ISSUE TITLE
MRR is lower than the base model using Tensorflow

### ISSUE THREAD

**Issue by suxnju** [bentrevett/CodeSearchNet#1]
Hi，
Thanks for your CodeSearchNet Code in Pytorch,it helps me a lot when I try to reproduce the results using Pytorch.
But I cannot get the similar results.For example,testing Python language using bow model and cosine loss,the mrr is 0.33(<0.61).Have you ever been in the same situation?
